### ISSUE TITLE
Does the current code work?

### ISSUE THREAD

**Issue by liddad** [vicktor/FreeStyleLibre-NFC-Reader#14]
Hey, I'm just wondering, does this version of the code work? I'm just comparing it to this https://github.com/JoernL/LimiTTer/blob/master/LimiTTer.ino and there seems to be some minor differences.

**Comment by 4gra** [comment]
It worked for me some months ago, for some definition of the word (I made a few modifications to my own branch to support an older version of Android and newer tools): it scanned successfully and gave a live (raw) reading decoded from the data. Perhaps you could be more specific about the differences between these different pieces of software; it might be instructive if you have a suspicion that one might be wrong - but note they're designed to fulfil very different purposes so some difference should surely be expected.

**Comment by liddad** [comment]
Hi! I have my own code working now, I used this as a reference to figure out how the Libre sensor works! 
For anyone who finds this in the future: It turns out the differences were due to 2 bytes that were returned by the NFC breakout board used in the LimiTTer code! The other difference was how the glucose readings were calculated but neither way matches the official LibreLink app's values.
Also this code reads blocks 3-39 where the LimiTTer code reads 3-15 and 39, but that's because the other blocks contain the historical trend data which the LimiTTer code was not using. (But block 39 contains the time since the sensor was started so that's used in both).
